pkg/integrations/cloudwatch_exporter/docs/template.md

```diff
@@ -0,0 +1,6 @@
+# Supported services in discovery jobs
+
+The following is a list of AWS services that are supported in `cloudwatch_exporter` discovery jobs. When configuring a
+discovery job, the `type` field of each `discovery_job` must match either one of the desired job namespace or alias.
+
+{{SERVICE_LIST}}
```

Comment:
Is this list going to increase that much in the future to need to have a tool to generate it automatically?

Reply:
Not that I know of, but we could eventually expand the tool to generate in the docs things such as cw dimensions exported in each discovery job type, and some other characteristics, which is a PITA to maintain by hand. And if we do add services, then this will catch all.